nml/global_constants.py

```diff
@@ -811,6 +804,9 @@ def constant_number(name, info, pos):
     "ANIM_TRIGGER_INDTILE_RECEIVED_CARGO"     : 3,
     "ANIM_TRIGGER_INDTILE_DISTRIBUTES_CARGO"  : 4,
 
+    "OBJ_ANIM_IS_BUILT"                     : 0,  # deprecated
+    "OBJ_ANIM_PERIODIC"                     : 1,  # deprecated
+    "OBJ_ANIM_SYNC"                         : 2,  # deprecated
```

Comment:
There's an integrated deprecation mechanism:
```suggestion
    "OBJ_ANIM_IS_BUILT"                     : "ANIM_TRIGGER_OBJ_BUILT",
    "OBJ_ANIM_PERIODIC"                     : "ANIM_TRIGGER_OBJ_TILELOOP",
    "OBJ_ANIM_SYNC"                         : "ANIM_TRIGGER_OBJ_256_TICKS",
```

Same applies to station anim.